cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -295,6 +295,11 @@ func NewManager(
 	console input.Console,
 	commandRunner exec.CommandRunner,
 ) (*Manager, error) {
+	// Use silent console when console is not provided
+	if console == nil {
+		console = input.NewSilentConsole()
+	}
```

Comment:
I'd avoid "unobvious" behavior if possible. Can the caller be the one that is passing in the desired `console` instance? Otherwise, this needs to be documented properly.

Reply:
I can do that, sure